src/sentry/web/api.py

```diff
@@ -145,19 +187,7 @@ def process_event(event_manager, project, key, remote_addr, helper, attachments)
     if rate_limit is None or rate_limit.is_limited:
         if rate_limit is None:
             api_logger.debug('Dropped event due to error with rate limiter')
-        tsdb.incr_multi(
-            [
-                (tsdb.models.project_total_received, project.id),
-                (tsdb.models.project_total_rejected, project.id),
-                (tsdb.models.organization_total_received,
-                    project.organization_id),
-                (tsdb.models.organization_total_rejected,
-                    project.organization_id),
-                (tsdb.models.key_total_received, key.id),
-                (tsdb.models.key_total_rejected, key.id),
-            ],
-            timestamp=tsdb_start_time,
-        )
+        track_outcome(project.organization_id, project.id, key.id, 'rate_limited', None)
```

Comment:
I feel like theres a rate_limit reason that we track somewhere based on which quota returned `is_limited`.